packages/webapp/pages/sources/index.tsx

```diff
@@ -48,6 +51,7 @@ const SourceTopList = ({
           <ListItem key={item.id} index={i + 1} href={item.permalink}>
             <UserHighlight
               {...item}
+              userType={UserType.Source}
```

Comment:
This does not come from item?

Reply:
Not really, removing this line make the component broke for ts check :(

Reply:
`userType` was optional before and `UserHighlight` had it default to `UserType.Source`, I am actually more in favour to keeping this 👍